File: ExamTest/app/src/main/java/ru/project/examtest/data/db/DataBaseNames.java
package ru.project.examtest.data.db;

import android.provider.BaseColumns;

public class DataBaseNames {
    public static final String database_name = "database.db";
    public static final int database_version = 1;
    public static final String table_name_teachers = "teachers";
    public static final String table_username_teachers = "username";
    public static final String table_password_teachers = "password";

    public static final String SQL_CREATE_TABLE_TEACHERS = "CREATE TABLE "+ table_name_teachers +
            "(" +
            BaseColumns._ID + " INTEGER PRIMARY KEY," +
            table_username_teachers + " TEXT," +
            table_password_teachers + " TEXT" +
            ")";

    public static final String SQL_DELETE_TABLE_TEACHERS = "DROP TABLE IF EXISTS "
            + table_name_teachers;
}

File: MyAppNotes/app/src/main/java/codes/myappnotes/db/DataBaseNames.kt
package codes.myappnotes.db

import android.provider.BaseColumns

object DataBaseNames: BaseColumns {
    const val DATABASE_NAME = "database_notes.db"
    const val TABLE_NAME = "notes"
    const val TABLE_HEADER = "header"
    const val TABLE_TEXT = "text"
    const val DATABASE_VERSION = 1

    const val SQL_CREATE_TABLE = "CREATE TABLE $TABLE_NAME(" +
            "${BaseColumns._ID} INTEGER PRIMARY KEY," +
            "$TABLE_HEADER TEXT NOT NULL," +
            "$TABLE_TEXT TEXT NOT NULL)"

    const val SQL_DELETE_TABLE = "DROP TABLE $TABLE_NAME"

}
